cdn/dev/keyboard-search/install.css

```diff
@@ -0,0 +1,15 @@
+/* List entries */
```

Comment:
This numbered list pattern is not really present, yet. It may be removed in future.